src/services/service.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace MyApp\services;
+
+interface Service
```

Comment:
What is the purpose of the service?
The name of the service must be specific. Like

```
BookingServiceInterface
PaymentServiceInterface
```